bcgov/HOUS-permit-portal
# Add contact edit and delete modal and related functionality

## Description

Add contact edit and delete


## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature

## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-1345

## Steps to QA

See card.
Submitter can click contact autofill on contact inputs, and now has an "edit" link on each contact.
Can use this to open the modal where the user can edit and delete contact.
